src/components/CharacterDetailCard/styles.scss

```diff
@@ -0,0 +1,42 @@
+.container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 16px;
+    gap: 10px;
+
+    width: 368px;
+    height: 81px;
```

Comment:
I couldn't be sure about these two. 
Do you need to set the height? 
For width maybe max-width can be used here.